astroid/as_string.py

```diff
@@ -591,6 +606,80 @@ def visit_starred(self, node):
         """return Starred node as string"""
         return "*" + node.value.accept(self)
 
+    def visit_match(self, node: "Match") -> str:
```

Comment:
The `Match` nodes are only imported with `if TYPE_CHECKING` to prevent a circular dependency. Since we can't use postponed evaluation of annotations just yet (requires 3.7), these need to be strings for now.

Reply:
Yes of course. I forgot the 'if' statement!